sound/soc/sof/amd/acp-ipc.c

```diff
@@ -200,6 +202,22 @@ irqreturn_t acp_sof_ipc_irq_thread(int irq, void *context)
 		return IRQ_HANDLED;
 	}
 
+	if (desc->probe_reg_offset) {
+		u32 val;
+		u64 posn;
+
+		val = snd_sof_dsp_read(sdev, ACP_DSP_BAR, desc->probe_reg_offset);
+		if (val & BIT(31)) {
+			posn = val & ~BIT(31);
```

Comment:
maybe use a define for the magic 31, and explain what this test does?

Reply:
done